ACE/bin/make_release.py

```diff
@@ -461,8 +467,8 @@ def update_latest_branch (product, which):
 
     name = "Latest_" + which
 
-    vprint ('Fast-forwarding', name, 'to master')
-    ex ("cd $DOC_ROOT/" + product + " && git fetch . master:" + name)
+    vprint ('Fast-forwarding', name, 'to', opts.branch)
```

Comment:
More places where a space is missing in the output 

``` suggestion
    vprint ('Fast-forwarding', name, 'to ', opts.branch)
```

Reply:
This function passes its arguments to Python's builtin `print`, which adds spaces between arguments by default.